### ISSUE TITLE
[ENHANCEMENT] Provide a way to have init be triggered on headless nodes

### ISSUE THREAD

**Issue by matthme** [holochain/holochain#3123]
Currently, the init callbacks get triggered lazily with the first zome call in a dna. This is a problem in the case of headless nodes as there is no client to make that first zome call and they nevertheless often require the init callback, for example to advertise themselves in the network as participants that accept remote calls.

**Comment by ThetaSinner** [comment]
If we had zome calls available in the `hc` command, then this might be a good way to initialise. Then there's a possibility of automating this

**Comment by mattyg** [comment]
Proposal:

Since a zome already has an "init" function, this can be called directly to trigger initialization.

The problem is, when a zome's `init` function is called directly, the conductor still runs `init` before executing the `init` call. Instead it should only call `init` once.

**Comment by ThetaSinner** [comment]
Agreed, I'll start with the `init` behavior in Holochain and then figure out the client stuff
